integration-tests/docker/test_env/test_env_builder.go

```diff
@@ -46,6 +48,32 @@ func NewCLTestEnvBuilder() *CLTestEnvBuilder {
 	}
 }
 
+func (b *CLTestEnvBuilder) WithTestEnv(te *CLClusterTestEnv) (*CLTestEnvBuilder, error) {
```

Comment:
@tateexon what this function does? Creates builder from config file? If yes, maybe it should not be `CLTestEnvBuilder` method but a normal function? And maybe a better name could be sth like "NewTestEnvBuilderFromConfig()`?

Reply:
Will add a comment to the function but I intend it to be used just like any of the other With commands except this one needs some error handling. We had handling of the CLClusterTestEnv configuration spread out and unable to easily be changed and used outside of EVM use cases so this just bundles it up into one place and exposes it as part of the builder chain.

Reply:
Example usage: https://github.com/smartcontractkit/chainlink-solana/pull/545/files#diff-84d6c8bb564dc0897c30ad0c4dc9ecb05db8ce2b220f64b5f5618ca3f8b8fa4aR187-R194